src/app/services/tests/GameService.spec.ts

```diff
@@ -183,6 +207,60 @@ describe('GameService', () => {
         expect(chatDAO.set).toHaveBeenCalledOnceWith('partId', {});
         expect(configRoomDAO.set).toHaveBeenCalledOnceWith('partId', configRoom);
     }));
+    describe('resign', () => {
+        it('should delegate to updateAndBumpIndex with a MGPResult.RESIGN value', fakeAsync(async() => {
```

Comment:
j'suis plutôt pour rendre updateAndBumpIndex privé, et avoir un gameService.resign qui fait la délégation, qu'en penses-tu ?

Reply:
updateAndBumpIndex is dead ! test supprimé !